gfx.h

```diff
@@ -454,6 +454,7 @@ class GfxInternal
     uint32_t max_frames_in_flight_ = 0;
 
     ID3D12Device *device_ = nullptr;
+    IDXGIAdapter1 *adapter_ = nullptr;
```

Comment:
```D3D12MA::CreateAllocator``` now requires that an adapter also be passed along with the device so I added the adapter as an internal variable

Reply:
Sounds good.